app/src/main/java/com/github/neighbortrader/foodboardapp/handler/clientmodelHandler/UserHandler.java

```diff
@@ -0,0 +1,114 @@
+package com.github.neighbortrader.foodboardapp.handler.clientmodelHandler;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.github.neighbortrader.foodboardapp.clientmodel.Address;
+import com.github.neighbortrader.foodboardapp.clientmodel.User;
+import com.github.neighbortrader.foodboardapp.handler.contextHandler.ContextHandler;
+import com.github.neighbortrader.foodboardapp.handler.gsonHandler.GsonHandler;
+import com.github.neighbortrader.foodboardapp.handler.requestsHandler.OnRequestEventListener;
+import com.github.neighbortrader.foodboardapp.handler.requestsHandler.RequestTyps;
+import com.github.neighbortrader.foodboardapp.handler.requestsHandler.UserRequestHandler;
+import com.google.gson.Gson;
+
+import java.util.UUID;
+
+import static android.content.Context.MODE_PRIVATE;
+
+public class UserHandler {
+
+    public static final String TAG = UserHandler.class.getSimpleName();
+
+    private static User userInstance;
+
+    private static final String SHARED_PREFERENCES_FILE_USER_INFO = "userInfo";
```

Comment:
wos is des ?

Reply:
Was... Kommentier mal bissel genauer kann keiner Nachvollziehen. Wenn du Shared Pref. meinst dann Google